packages/ui-webpack-config/config/plugins.js

```diff
@@ -40,6 +38,8 @@ const envVars = {
 }
 
 module.exports = [
-  new webpack.EnvironmentPlugin(envVars),
-  DEBUG ? new WatchMissingNodeModulesPlugin(path.resolve('node_modules')) : null
-].filter(Boolean)
+  // fix Buffer/process is not defined errors:
+  new webpack.ProvidePlugin({ process: 'process/browser' }),
```

Comment:
Can't we simply call this only one time?
```js
new webpack.ProvidePlugin({
    process: 'process/browser',
    Buffer: ['buffer', 'Buffer']
  }),
```

Reply:
yes, done